static/src/js/components/GranuleResults/__tests__/GranuleResultsItem.test.js

```diff
@@ -335,316 +363,417 @@ function setup(type, overrideProps) {
-
-      const downloadButton = enzymeWrapper.find(GranuleResultsDataLinksButton)
+      setup('cmr-no-download')
 
-      expect(downloadButton.length).toEqual(0)
+      expect(screen.queryAllByLabelText('Download single granule data').length).toEqual(0)
     })
   })
 
   describe('without an granuleThumbnail', () => {
     test('does not render an granuleThumbnail', () => {
-      const { enzymeWrapper } = setup('no-thumb')
+      setup('no-thumb')
 
-      expect(enzymeWrapper.find('.granule-results-item__thumb').length).toEqual(0)
+      expect(screen.queryByLabelText('Link to granule')).not.toBeInTheDocument()
+      expect(screen.queryByLabelText('Granule thumbnail image')).not.toBeInTheDocument()
     })
 
     test('does not add the modifier class name', () => {
-      const { enzymeWrapper } = setup('no-thumb')
+      setup('no-thumb')
 
-      expect(enzymeWrapper.props().className).not.toContain('granule-results-item--has-thumbnail')
+      expect(screen.getByTestId('granule-results-item').className).not.toContain('granule-results-item--has-thumbnail')
     })
   })
 
   describe('with a granuleThumbnail', () => {
     test('without a full size browse', () => {
-      const { enzymeWrapper } = setup('cmr')
+      const { props } = setup('cmr')
 
-      expect(enzymeWrapper.find('.granule-results-item__thumb').length).toEqual(1)
-      expect(enzymeWrapper.find('.granule-results-item__thumb').type()).toEqual('div')
+      const { granule } = props
+
+      const { title, granuleThumbnail } = granule
+
+      expect(EDSCImage).toHaveBeenCalledTimes(1)
+      expect(EDSCImage).toHaveBeenCalledWith(
+        expect.objectContaining(
+          {
+            alt: `Browse Image for ${title}`,
+            className: 'granule-results-item__thumb-image',
+            height: 85,
+            isBase64Image: true,
+            src: granuleThumbnail,
+            width: 85,
+            useSpinner: false
+          }
+        ),
+        {}
+      )
+
+      expect(screen.queryByLabelText('Link to granule')).not.toBeInTheDocument()
     })
 
     test('with a full size browse', () => {
-      const { enzymeWrapper } = setup('with-browse')
-      expect(enzymeWrapper.find('.granule-results-item__thumb').length).toEqual(1)
-      expect(enzymeWrapper.find('.granule-results-item__thumb').type()).toEqual('a')
+      const { props } = setup('with-browse')
+      const { granule } = props
+
+      const { browseUrl } = granule
+
+      expect(screen.getByLabelText('Link to granule')).toBeInTheDocument()
+      expect(screen.getByLabelText('Link to granule').href).toEqual(browseUrl)
     })
 
     test('adds the modifier class name', () => {
-      const { enzymeWrapper } = setup('with-browse')
+      setup('with-browse')
 
-      expect(enzymeWrapper.props().className).toContain('granule-results-item--has-thumbnail')
+      expect(screen.getByTestId('granule-results-item').className).toContain('granule-results-item--has-thumbnail')
```

Comment:
Avoid using test ids, need to find a way to find this element in a way that better simulates how a user would use the app